### PR TITLE
Add my first file

### PR DESCRIPTION
**Pull request by RagunathanDPI**
Add my first file

### CODE REVIEW

**Comment by yaroff** [comment]
@RagunathanDPI Nice work! What is this city?

File: RaguanthanDPI.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+githubHandle: RagunathanDPI
```
Comment (crichID) [inline RaguanthanDPI.json]: @RagunathanDPI you will need to add three dashes to the beginning and end of your file so it looks like this:

```
---
githubHandle: RagunathanDPI
latitude: 37.751
longitude: -97.822
---
```

File: RaguanthanDPI.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+githubHandle: RagunathanDPI 
+latitude: 37.751      
+longitude: -97.822E
```
Comment (crichID) [inline RaguanthanDPI.json]: @RagunathanDPI you will need to remove the E from the end of this line!

**Comment by crichID** [comment]
@RagunathanDPI 
I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/6646/commits/287e24486f92e8f3e6daec791d031c794b284d0d. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by May 26th, I will go ahead and do it just to keep the repository clean!

**Pull request merged**